taichi/python/export_lang.cpp

```diff
@@ -266,8 +267,16 @@ void export_lang(py::module &m) {
       .def("begin_frontend_if_true", &ASTBuilder::begin_frontend_if_true)
       .def("pop_scope", &ASTBuilder::pop_scope)
       .def("begin_frontend_if_false", &ASTBuilder::begin_frontend_if_false)
-      .def("insert_deactivate", Deactivate)
-      .def("insert_activate", Activate)
+      .def("insert_deactivate",
+           [](ASTBuilder *ast_builder, SNode *snode,
+              const ExprGroup &expr_group) {
```

Comment:
We'd better not directly write functions in `export_lang`. Put them inside `ASTBuilder`.